AdminWebsite/AdminWebsite/ClientApp/src/app/suitability/services/suitability-answer.mapper.ts

```diff
@@ -0,0 +1,25 @@
+export const IndividualQuestionKeys = {
+  AboutYou: 'ABOUT_YOU',
+  Consent: 'CONSENT',
+  Room: 'ROOM',
+  Internet: 'INTERNET',
+  Interpreter: 'INTERPRETER',
+  Computer: 'COMPUTER',
+  Camera: 'CAMERA_MICROPHONE'
+};
+
+export const RepresentativeQuestionKeys = {
+  AboutYou: 'ABOUT_YOU',
+  AboutYourClient: 'ABOUT_YOUR_CLIENT',
+  ClientAttendance: 'CLIENT_ATTENDANCE',
+  HearingSuitability: 'HEARING_SUITABILITY',
+  Room: 'ROOM',
+  Camera: 'CAMERA_MICROPHONE',
+  Computer: 'COMPUTER'
+};
+
+export const SelfTestQuestionKeys = {
+  SeeYourself: 'SEE_YOURSELF',
```

Comment:
Looks good, I wonder if we should prefix them as well so we know they're part of the self-test

The runway team usually talks about the self-test in terms of "kit check", so maybe:
```
export const SelfTestQuestionKeys = {
SeeYourself: 'KIT_SEE_YOURSELF',
SeeYourself: 'KIT_MICROPHONE', // I think we're testing the mic here rather than the speakers?
SeeYourself: 'KIT_SEE_HEAR_CLEARLY',
}
```